### ISSUE TITLE
merge in from avalon

### ISSUE THREAD

**Issue by hannesdelbeke** [pyblish/pyblish-krita#1]
@BigRoy are you happy if we merge in your work from avalon in this repo?
i'm happy to do some legwork
https://github.com/getavalon/core/issues/496

**Comment by BigRoy** [comment]
Absolutely, not a problem. Be aware that it was purely a prototype and I don't use Krita myself so I am not sure how it (still) works.
